deploy/ray-operator/utils/util.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"strings"
+)
+
+// The function isCreated returns true if pod has been created and is maintained by the API server
+func IsCreated(pod *corev1.Pod) bool {
```

Comment:
```suggestion
func isCreated(pod *corev1.Pod) bool {
```

Reply:
In Go, capital letter means the function can be accessed by other package, lower-case means private can only be accessed in the same package